src/ForkCMS/Utility/Module/CopyContentToOtherLocale/Results.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace ForkCMS\Utility\Module\CopyContentToOtherLocale;
+
+use ForkCMS\Utility\Module\CopyContentToOtherLocale\Exception\CopiedModuleContentToOtherLocaleNotFound;
+
+final class Results
+{
+    /** @var array */
+    private $idMap;
+
+    /** @var array */
+    private $moduleExtraIdMap;
+
+    public function add(string $moduleName, array $idMap, array $moduleExtraIdMap): void
+    {
+        $this->idMap[$moduleName] = $idMap;
+        $this->moduleExtraIdMap[$moduleName] = $moduleExtraIdMap;
+    }
+
+    /**
+     * @param string $moduleName
+     * @param mixed $id
+     * @return mixed - The new id
+     * @throws \Exception
+     */
+    public function getModuleExtraId(string $moduleName, $id)
```

Comment:
isn't the id an int? I don't think module extra's allow strings there

Reply:
Already changed